client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,66 @@
+.scoreboard > div.scoreboardIntermediate {
+  position: relative;
+  z-index: 0;
+  border: 2px solid black;
+  box-sizing: border-box;
+  background: #fff;
+  border-radius: 8px;
+  overflow: auto;
+  height: 100%;
+  width: 100%;
+}
+
+.scoreboard > div.scoreboardIntermediate > table {
+  border-collapse: separate;
+  border-spacing: 0;
+  width: 100%;
+}
+
+.scoreboard > div.scoreboardIntermediate > table tbody td {
+  border-bottom: 1px solid #ddd;
+  border-right: 1px solid #ddd;
+  color: #333;
+  padding: 5px;
+  text-align: right;
+  white-space: nowrap;
+  background-color: #f8f8f8;
```

Comment:
I don't think this background should be specified on the cells - it's practically the same color as the scoreboard one. Instead we should only have a background specified on the root widget which can be overridden by the game author.

Reply:
My problem was `border-radius` + `overflow: auto`. If the background is on the main widget, the border-radius has to be there as well. And so does the overflow; otherwise it breaks the rounded corners at least on Firefox IIRC.

But `overflow: auto` on the widget itself hides any child widgets. If you have a solution for that, I agree that your suggestion is the better approach.

Reply:
Are you sure? border-radius seems to affect the border and background-color painting on both firefox and chrome without overflow for me: https://jsbin.com/fizejub/edit?html,css,output

Reply:
But it doesn't clip the children for me:

![image](https://user-images.githubusercontent.com/73538315/209982104-a060f314-31fa-4aa4-bd33-45333d360a15.png)

Reply:
Although it does for the intermediate div so I probably did something wrong.

Reply:
Right, you need to add `border-radius: inherit` to `.scoreboardIntermediate` so that it inherits the parent widget's border radius. This will also allow using the border radius property on the widget to update the scoreboard i think.

Reply:
I'll give it another shot. If you have the time, just commit the changes yourself. You seem to have a good idea.

Reply:
Sure, will do!

Reply:
Done.

Ideally the shaded cells would also use partial opacity so that they would work well against more colors, but they also have to be opaque in order to occlude the cells they're covering for the sticky scroll behavior. We can use `background: inherit` to re-apply the root background to those cells but then we'd need a different box or property to apply `rgba(0, 0, 0, 0.14)` to in order to add the shading.